projects/docs-tools/src/modules/demo-page/demo-page.component.ts

```diff
@@ -74,7 +96,30 @@ export class SkyDocsDemoPageComponent
    * Specifies the local path to the module's source code. The value is relative to the root directory.
    */
   @Input()
-  public moduleSourceCodePath: string;
+  public set moduleSourceCodePath(path: string | undefined) {
+    this.#_moduleSourceCodePath = path;
+    this.testingSourceCodePath = path.replace('modules', 'testing');
```

Comment:
I don't think we should rely on a folder named "modules" in the source code path. Also, this regular expression isn't specific enough to only find a directory (it will replace the first occurrence of the word "modules").

Could we have a dedicated `testingSourceCodePath` input to avoid the problems above?

Reply:
Updated to look for a directory called `modules` and accounted for both forward and backslashes.